File: benchmark/results/python-invocation-script/run-003/a-mcp/output/ordering/services/order_service.py
from ordering.aggregates.order import Order

"""OrderService service."""
class OrderService:
    def create(self, input: Order) -> Order:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Order | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[Order]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/ordering/services/OrderService.java
package com.metaengine.demo.ordering.services;

import com.metaengine.demo.ordering.aggregates.Order;
import java.util.List;

/** Application service exposing order operations. */
public class OrderService {
    /** Create an order from the provided partial input. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an order by id. Returns null when not found. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} orders. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an order by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
